cmd/poseidon/poseidon.go

```diff
@@ -121,11 +121,14 @@ func main() {
 		panic(err)
 	}
 	defer conn.Close()
-	// Check if frimament grpc service is available and then proceed
+	// Check if firmament grpc service is available and then proceed
 	WaitForFirmamentSerive(fc)
 	go schedule(fc)
 	go stats.StartgRPCStatsServer(statsServerAddress, firmamentAddress)
 	kubeVer := strings.Split(kubeVersion, ".")
+	if len(kubeVer) < 2 {
+		glog.Fatalf("Incorrect content in --kubeVersion %s", kubeVersion)
```

Comment:
add hint message? E.g. kubeVersion should be in the format of X.Y?

Reply:
Done